Shadem/Source/Renderer/CMRenderer.cpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Renderer/CMRenderer.h"
+
+CMRenderer::CMRenderer()
+{
+}
+
+void CMRenderer::draw(Object* obj, Scene& scene)
+{
+	obj->draw(*scene.getCamera());
```

Comment:
Chyba zwróciłem na to uwagę ostatnio, ale warto się jeszcze zastanowić czy nie lepiej będzie od razu przekazywać wskaźnika na kamerę, zamiast zawsze przekazywać referencje do sceny i dopiero potem za każdym razem wyciągać wskaźnik